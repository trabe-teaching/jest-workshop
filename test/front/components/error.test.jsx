import React from "react";
import { render } from "enzyme";
import Error from "components/error";

describe("front/components/error", () => {
  it("renders nothing if not visible", () => {
    expect(render(<Error visible={false} />)).toMatchSnapshot();
  });

  it("renders if visible", () => {
    expect(render(<Error visible />)).toMatchSnapshot();
  });
});
