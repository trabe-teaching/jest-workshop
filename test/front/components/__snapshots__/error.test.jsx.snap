// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`front/components/error renders if visible 1`] = `
<div
  class="modal is-active"
>
  <div
    class="modal-background"
  />
  <div
    class="modal-content"
  >
    <article
      class="message is-danger"
    >
      <div
        class="message-header"
      >
        <p>
          The death star has exploded
        </p>
      </div>
      <div
        class="message-body"
      >
        <figure
          class="image"
        >
          <img
            alt="Image"
            src="assets/death-star.png"
          />
        </figure>
      </div>
    </article>
  </div>
</div>
`;

exports[`front/components/error renders nothing if not visible 1`] = `null`;
